vscode/src/debugger/session.ts

```diff
@@ -48,6 +49,7 @@ export class QscDebugSession extends LoggingDebugSession {
   ) {
     super();
 
+    this.program = vscode.Uri.parse(this.config.program).path;
```

Comment:
Did you test this with a virtual file system? I'm not sure path gives you the protocol. I'd have thought you'd want the full Uri string to load the program.